TriforkSwiftExtensions/TriforkSwiftExtensionsTests/UIViewExtensionTests.swift

```diff
@@ -43,4 +43,22 @@ class UIViewExtensionTests: XCTestCase {
         XCTAssertEqual(parentMany.subviews.count, 0)
         
     }
+    
+    func testHide() {
+        let view = UIView()
+        view.isHidden = false
+        view.hide()
+        XCTAssert(view.isHidden == true, "view should be hidden when calling hide on a visible view")
+        view.hide()
+        XCTAssert(view.isHidden == true, "nothing should happen second time you call hide")
+    }
+    
+    func testShow() {
+        let view = UIView()
+        view.isHidden = true
+        view.show()
+        XCTAssert(view.isHidden == false, "view should be visible(not hidden) when calling show on an invisible view")
```

Comment:
Please use `XCTAssertFalse` 👀

Reply:
This is a good example of why I hate style linting. The code @ktvtrifork wrote is perfectly fine and does exactly the same as you suggest, @kdvtrifork. It is a matter of taste whether `XCTAssert` and `XCTAssertFalse` is the right choice. I don't think we should use the reviews for that kind of stuff - just my opinion 🤷🏻‍♂️

Reply:
It does the same yes. There is nothing there. Just style. 
But i still think we need to keep a style in the project for readability. 

That's why I prefer a linter, so we are consistent in the code 😉

But let's close and discuss this when we play pool at work 🎱

Reply:
I Would actually side with @kdvtrifork  here @tkctrifork ; 
Or more precise IMO that the way to write XCTAssertions are rather dump; 
(I much more preffere the cleanness of something like  
```swift
view.isHidden.assert(false, "the error message here")
//or for that matter
view.isHidden.assertFalse("the error message here")
```
see for example 
https://github.com/Tvede-dk/commonsense-swift/blob/master/csenseSwiftTests/datastructures/SortedArrayTests.swift (I find the style a lot simpler to read, and write, especially since all is not wrapped nor nested in XCTAssert***)